File: app/assets/javascripts/data/player.ts
import {Club} from "./club"

export class Player {
	id: number;
	firstName: string;
	lastName: string;
	ttr: number;
	sex: string;
	club: Club;

	constructor(firstName: string, lastName:string, club: Club){
		this.firstName = firstName;
		this.lastName = lastName;
		this.club = club;
	}
}

File: management-backend/src/main/kotlin/de/ttt/management/registration/domain/player/Player.kt
package de.ttt.management.registration.domain.player

class Player(
    var id: Long? = null,
    var firstName: String? = null,
    var lastName: String? = null,
    var email: String? = null,
    var telNr: String? = null,
    var location: String? = null,
    var street: String? = null,
    var plz: String? = null,
    var sex: String? = null,
    var ttr: Int? = null,
    var club: Club? = null,
    var paid: Boolean = false
)
